docs/blog/index.html

```diff
@@ -4,7 +4,7 @@
 <meta charset="UTF-8">
 <!--[if IE]><meta http-equiv="X-UA-Compatible" content="IE=edge"><![endif]-->
 <meta name="viewport" content="width=device-width, initial-scale=1.0">
-<meta name="generator" content="Asciidoctor 1.5.4">
+<meta name="generator" content="Asciidoctor 1.5.5">
 <title>Bloomberg announcing BuckleScript 1.0</title>
```

Comment:
can you not check in generated html? it may result in some inconsistency if we have different versions of asciidoctor